WireAPI/Package.swift

```diff
@@ -48,14 +46,18 @@ let package = Package(
                 .process("APIs/UserPropertiesAPI/Resources"),
                 .process("APIs/SelfUserAPI/Resources"),
                 .process("Network/PushChannel/Resources")
-            ],
-            swiftSettings: swiftSettings
+            ]
         )
     ]
 )
 
-let swiftSettings: [SwiftSetting] = [
-    .enableUpcomingFeature("ExistentialAny"),
-    .enableUpcomingFeature("GlobalConcurrency"),
-    .enableExperimentalFeature("StrictConcurrency")
-]
+for target in package.targets {
```

Comment:
I didn't realize how much we can manipulate packages!

Reply:
I wasn't aware neither, since the code completion somehow doesn't work well in Package.swift.